src/compiler/crystal/codegen/codegen.cr

```diff
@@ -123,7 +123,7 @@ module Crystal
     property last : LLVM::Value
 
     class LLVMVar
-      getter pointer : LLVM::Value
+      property pointer : LLVM::Value
```

Comment:
Seems like this change can be reverted.

Reply:
Thank you! IT was needed originally but I did refactored the code so now it is redundant. I reverted it back to getter.

Reply:
I fixed it on my side but will wait if you will find anything else that I missed and will recheck my changes as well.